core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/localscan/LocalCsvFileScanOpDesc.java

```diff
@@ -86,17 +82,102 @@ public Schema sourceSchema() {
     }
 
     private Schema inferSchema(String headerLine) {
+        java.util.List<AttributeType> attributeType = IntStream
```

Comment:
1. name the variable as plural form since it is a collection.
2. this code seems only initializing a `List<AttributeType>` of `AttributeType.INTEGER` types. is there a reason we by default initialize it to Integer? why can't we use an empty list?
3. please import `java.lang.List` global.

Reply:
1. change the name to "attributeTypeList"
2. Initializing it with `AttributeType.INTEGER` can start infer with INTEGER, because it is the lowest priority in the Priority of the attributeType
3. "import scala.collection.immutable.List;" -> There is already one List which is immutable. I want to use the mutable List. I don't know how to solve this.